include/core/thermal_conductivity_model.h

```diff
@@ -188,4 +188,122 @@ class ThermalConductivityLinear : public ThermalConductivityModel
   const double B;
 };
 
+/**
+ * @brief ThermalConductivityPhaseChange Implements a phase-dependant thermal conductivity
+ */
+class ThermalConductivityPhaseChange : public ThermalConductivityModel
+{
+public:
+  /**
+   * @brief Default constructor
+   */
+  ThermalConductivityPhaseChange(const double thermal_conductivity_s,
```

Comment:
It would be a better idea to pass the phase change parameters as a whole const Parameters::PhaseChange
This way if there are any extensions or etc. to them, we can leverage them more easily and it is less confusing.

Reply:
Done.